ee/tabby-ui/README.md

```diff
@@ -1,3 +1,42 @@
 # Tabby UI
 
-Run `yarn dev` to start the development server, and use `TABBY_FRONTEND_URI=localhost:3000 cargo ...` to start tabby server with the development frontend.
+## 🤝 Contribuing
+Thank you for your interest in contributing to Tabby! We appreciate all contributions. For a better experience and support, join us on [Slack](https://links.tabbyml.com/join-slack)!
+
+### Local Setup
+Full guide at [CONTRIBUTING.md](https://github.com/TabbyML/tabby/blob/main/CONTRIBUTING.md#local-setup)
+
+### Get the Code
+To begin contributing to Tabby, first clone the repository locally:
+
+```
+git clone --recurse-submodules https://github.com/TabbyML/tabby
+```
+
+If you have already cloned the repository, you could run the git submodule update --recursive --init command to fetch all submodules.
+
+### Running
```

Comment:
Please provide higher level overview on how caddy orchestrate tabby-ui / local tabby together, and explain how tabby-ui is embed in tabby binary in production distribution